### PR TITLE
Add Manual Testing for edit command in DG

### LINKED ISSUE #284: [PE-D][Tester A] DG Instruction for manual testing
Just wanted to check if `{more test cases}` was left there intentionally.

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: Fluffykan/ped#8

### PR DESCRIPTION
**Pull request by colinhia**
Closes #284

### CODE REVIEW

**Review by FionaQY — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -697,6 +697,29 @@ testers are expected to do more *exploratory* testing.
...
    2. Functions similar to above example except that the indexes to be used are based on the new list shown.
 
+### Editing a client
+
+1. Editing a client while all clients are being shown
+
+    1. Prerequisites: List all clients using the `list` command. At least one client in the list.
+    2. Test case: `edit 1 n/ <NAME>`<br>
+         Expected: The name of the client in index 1 will be changed to <NAME>.
+    3. Test case: `edit 1 n/ <NAME> p/<PHONE>`<br>
```
Comment (ZShunRen) [inline docs/DeveloperGuide.md:707]:
```suggestion
    3. Test case: `edit 1 n/ <NAME> p/ <PHONE>`<br>
```

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -697,6 +697,29 @@ testers are expected to do more *exploratory* testing.
...
+         Expected: The name and the phone of the client in index 1 will be changed to <NAME> and <PHONE> respectively.
+   4. Other correct edit commands to try: `edit 1 n/ <NAME> p/<PHONE> e/ <EMAIL>`, `...` (for any number of valid flags and valid arguments)<br>
```
Comment (ZShunRen) [inline docs/DeveloperGuide.md:709]:
```suggestion
   4. Other correct edit commands to try: `edit 1 n/ <NAME> p/ <PHONE> e/ <EMAIL>`, `...` (for any number of valid flags and valid arguments)<br>
```

**Review by ZShunRen — APPROVED** [review]
LGTM, just make the following changes.

**Pull request merged**